model/atmosphere/dycore/src/icon4py/model/atmosphere/dycore/stencils/compute_advection_in_vertical_momentum_equation.py

```diff
@@ -115,15 +112,11 @@ def _compute_advection_in_vertical_momentum_equation(
     cell_upper_bound: gtx.int32,
     nlev: gtx.int32,
     nrdmax: gtx.int32,
-    start_cell_lateral_boundary: gtx.int32,
-    end_cell_halo: gtx.int32,
 ) -> tuple[fa.CellKField[ta.vpfloat], fa.CellKField[ta.vpfloat]]:
-    contravariant_corrected_w_at_cells_on_model_levels = where(
-        start_cell_lateral_boundary <= cell < end_cell_halo,
```

Comment:
This one can always be removed, will be same bounds as when calling program in vel adv.